src/main/java/com/flansmod/common/eventhandlers/BulletLockOnEvent.java

```diff
@@ -0,0 +1,34 @@
+package com.flansmod.common.eventhandlers;
+
+import com.flansmod.common.guns.EntityBullet;
+
+import cpw.mods.fml.common.eventhandler.Cancelable;
+import cpw.mods.fml.common.eventhandler.Event;
+import net.minecraft.entity.Entity;
+import net.minecraft.item.ItemStack;
+
+@Cancelable
+public class BulletLockOnEvent extends Event {
```

Comment:
Same for this event, it could use some documentation (i.e. when it's fired, what happens if it's cancelled, etc.).